workflow/scripts/plot-3prime-qc-histogram.py

```diff
@@ -0,0 +1,158 @@
+from turtle import title
```

Comment:
I thought you had merged this and the other script for the QC histogram. Did you just forget to delete one of them?

Reply:
Yes. Now we have only one script. I deleted the older script.